### PR TITLE
Enhancement/use enums

### PR DESCRIPTION
**Pull request by HenriChabert**
Fix for [ch62305]

### CODE REVIEW

File: python-lib/dku_deeplearning_image/misc_objects/dku_model.py [hunk 1]
```diff
@@ -174,10 +174,14 @@ def save_to_folder(self):
         logger.info("Model has been successfully saved.")
 
     def get_application(self):
-        dku_application_params = list(filter(lambda x: x['name'] == self.architecture, APPLICATIONS))
+        dku_application_params = [x for x in APPLICATIONS if x['name'].value == self.architecture]
```
Comment (alexbourret) [inline python-lib/dku_deeplearning_image/misc_objects/dku_model.py:177]: x...

File: python-lib/dku_deeplearning_image/misc_objects/dku_model.py [hunk 2]
```diff
@@ -174,10 +174,14 @@ def save_to_folder(self):
...
         if not dku_application_params:
-            available_apps = [x['name'] for x in APPLICATIONS]
-            raise IOError("The application you asked for is not available. Available are : {}.".format(available_apps))
+            available_apps = [x['name'].value for x in APPLICATIONS]
```
Comment (alexbourret) [inline python-lib/dku_deeplearning_image/misc_objects/dku_model.py:179]: Another x...

File: resource/helpers/runnable-helper.py [hunk 3]
```diff
@@ -32,7 +32,7 @@ def get_output_managed_folder(project_key):
 def get_model_choice():
     choices = [{
         'label': '{app[label]} trained on {ds}'.format(app=app, ds=ds.capitalize()),
-        'value': '{}::{}'.format(app['name'], ds)
+        'value': '{}::{}'.format(app['name'].value, ds)
```
Comment (alexbourret) [inline resource/helpers/runnable-helper.py:35]: Same here with variable `ds`

**Review by alexbourret — APPROVED** [review]
LGTM !

File: resource/helpers/runnable-helper.py (hunk 3 above)
Comment (HenriChabert) [reply resource/helpers/runnable-helper.py]: Fixed in b918723

File: python-lib/dku_deeplearning_image/misc_objects/dku_model.py (hunk 2 above)
Comment (HenriChabert) [reply python-lib/dku_deeplearning_image/misc_objects/dku_model.py]: Fixed in b918723

File: python-lib/dku_deeplearning_image/misc_objects/dku_model.py (hunk 1 above)
Comment (HenriChabert) [reply python-lib/dku_deeplearning_image/misc_objects/dku_model.py]: Fixed in b918723

**Pull request merged**